ExampleMod/Content/Items/Armor/ExampleHood.cs

```diff
@@ -0,0 +1,42 @@
+﻿using Terraria;
+using Terraria.ID;
+using Terraria.ModLoader;
+
+namespace ExampleMod.Content.Items.Armor
+{
+	// The AutoloadEquip attribute automatically attaches an equip texture to this item.
+	// Providing the EquipType.Head value here will result in TML expecting a X_Head.png file to be placed next to the item's main texture.
+	[AutoloadEquip(EquipType.Head)]
+	public class ExampleHood : ModItem
+	{
+		public override void SetStaticDefaults() {
+			Tooltip.SetDefault("This is a modded hood.");
+		}
+
+		public override void SetDefaults() {
+			Item.width = 18; // width of the item
```

Comment:
Not too important, but try to keep comments in the `// Comment`  format. In other words, just add capitalization.